packages/teams-js/src/internal/internalAPIs.ts

```diff
@@ -149,3 +149,17 @@ export function processAdditionalValidOrigins(validMessageOrigins: string[]): vo
   });
   GlobalVars.additionalValidOrigins = combinedOriginUrls;
 }
+
+/**
+ * @hidden
+ * Check if apiVersionTag developer sends follows the pattern starting with a lowercase 'v', then
+ * followed by one or more digits, then concated with underscore and some characters to indicate api name.
+ * For example, 'v2_app.getContext'. If yes, return true. Otherwise, return false.
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export function isFollowingApiVersionTagFormat(apiVersionTag: string): boolean {
```

Comment:
Would it make sense to also put this into the `telemetry.ts` file?

Reply:
Okay. I rethink of it and I think it worth to say we need to put everything related to apiVersionTag also to under telemetry.ts because that is it serves for. Will updated in next commit .